powertools-parameters/powertools-parameters-dynamodb/src/main/java/software/amazon/lambda/powertools/parameters/dynamodb/DynamoDbParamAspect.java

```diff
@@ -12,43 +12,45 @@
  *
  */
 
-package software.amazon.lambda.powertools.parameters.internal;
+package software.amazon.lambda.powertools.parameters.dynamodb;
 
 import org.aspectj.lang.ProceedingJoinPoint;
 import org.aspectj.lang.annotation.Around;
 import org.aspectj.lang.annotation.Aspect;
 import org.aspectj.lang.annotation.Pointcut;
 import org.aspectj.lang.reflect.FieldSignature;
-import software.amazon.lambda.powertools.parameters.BaseProvider;
-import software.amazon.lambda.powertools.parameters.Param;
-import software.amazon.lambda.powertools.parameters.ParamManager;
 
 @Aspect
-public class LambdaParametersAspect {
+public class DynamoDbParamAspect {
 
-    @Pointcut("get(* *) && @annotation(paramAnnotation)")
-    public void getParam(Param paramAnnotation) {
+
+    @Pointcut("get(* *) && @annotation(ddbConfigParam)")
```

Comment:
... But this also means we have some duplication of the aspect itself. I will make sure we have test coverage of each of these, as there is enough "compile time magic" that it's nice to cover that the aspect works.

Reply:
I don't know what is best here... 

Having different annotations means users will have to specify the good project in their weaving configuration in maven/gradle. We already have tons of questions/issues because of this.

But it's also probably cleaner (despite the duplication) for us and opens more doors (different config based on the store used). It's also maybe easier to read for users, more explicit: @DynamoDbParam, @SecretParam, ...

Let's go that way and document it / test it properly.

Reply:
I try to _bias for loose coupling_ when things get weird, and this was broken before - the annotation could only inject ~half of the parameter providers, and had weird coupling every which way. Agree that it needs good test coverage; weaving is error prone!